### DISCUSSION TITLE
Unknown protocol "https" in URI "https://myserver/71a2afab-5b32-4928-ad81-00caea4b7542.mp4"!

### DISCUSSION THREAD

**Discussion by ngivoni** [savonet/liquidsoap#2502] [User Support]
We are attempting to access a source file using https and receiving the following error:

` Unknown protocol "https" in URI "https://myserver/71a2afab-5b32-4928-ad81-00caea4b7542.mp4"!`

I've installed ssl on my ubuntu box but the error still persists.  Please advise

**Comment by toots** [comment]
Hi,

Most likely, you need to install the `curl` module:
```
% opam install ocurl
```
